app/views/posts/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="desktop">
+<% @posts.in_groups_of(3) do |post_group|%>
+<div class="row">
+<% post_group.each do |post|%>
+<div class="col"> 
+<div class="card">
+    <div class="card-body">
+        <h2><%= link_to post.title, post_path(post), class: "card-title" %></h2>
```

Comment:
@alecchanpascal Running into an error here. I think the problem is:
1. Seeding 50 posts
2. Grouping posts into 3 per row. 
3. Now there is a remainder of 2 so the last list is going to look like `[POST, nil, nil]`

I can give you an extra time to fix this error. Just reply to this comment once you push the fix.

![Screen Shot 2022-07-23 at 11 33 49 PM](https://user-images.githubusercontent.com/93061420/180635806-37a97270-010c-4525-b8b1-3958e5821758.png)

Reply:
I pushed the fix, hopefully everything should be working as intended now